src/control.cc

```diff
@@ -373,13 +378,17 @@ bool control_conn_t::process_start_stop(int pktType)
             if (service->get_state() == service_state_t::STARTED) ack_buf[0] = DINIT_RP_ALREADYSS;
             break;
         }
-        case DINIT_CP_RELEASESERVICE:
+        case control_cmd::CP_RELEASESERVICE:
             // remove required mark, stop if not required by dependents
             if (do_pin) service->pin_stop();
             service->stop(false);
             services->process_queues();
             if (service->get_state() == service_state_t::STOPPED) ack_buf[0] = DINIT_RP_ALREADYSS;
             break;
+        default:
+            // can't get here (hopefully)
+            // supreses compiler warning about unused  (-Wswitch)
```

Comment:
Please keep the comment to a minimum - since this is just for suppressing a warning, as little noise as possible is preferable. I suggest:
```
// avoid warning for unhandled values in switch/case
```

Reply:
done.